tests/h/views/api/users_test.py

```diff
@@ -110,7 +112,6 @@ def test_signs_up_user(self,
                            user_signup_service,
                            valid_payload):
         pyramid_request.json_body = valid_payload
-        user_signup_service.signup.return_value = factories.User.build(**valid_payload)
```

Comment:
What's the reason for removing `signup.return_value` here?

Reply:
Because a) it had no relevance on the test results and b) you can't build a `User` from a factory in a single go if the `**valid_payload` contains a list of `identities` dictionaries.